### PR TITLE
Change domain for authentication service

### PR DESCRIPTION
**Pull request by farski**

### CODE REVIEW

**Comment by chrisrhoden** [comment]
@farski thanks and sorry for hijacking.

@farski @eveadele can you make sure that my changes make sense to you? Some rambly commit messages there that should hopefully elucidate. Otherwise happy to go over them.

**Comment by chrisrhoden** [comment]
Gonna see what's up with travis.

**Comment by farski** [comment]
It ran before the dns changed

**Comment by chrisrhoden** [comment]
@farski rgr, deployed id.prx.org and trying some other things

File: lib/rack/prx_auth.rb [hunk 1]
```diff
@@ -1,34 +1,48 @@
...
   class PrxAuth
-    attr_reader :public_key
+    INVALID_TOKEN = [
+      401, {'Content-Type' => 'application/json'},
+      [{status: 401, error: 'Invalid JSON Web Token'}.to_json]
+    ]
+
+    DEFAULT_ISS = 'id.prx.org'
+
+    attr_reader :public_key, :issuer
```
Comment (eveadele) [inline lib/rack/prx_auth.rb]: Looks like the public_key instance variable doesn't exist anymore so we don't need an attr_reader for it.

**Comment by farski** [comment]
Aside from `Certificate#certificate` there's nothing I noticed, though I don't really know how JWT works so I couldn't review that very much.

**Comment by eveadele** [comment]
Does that mean I should merge?

**Comment by chrisrhoden** [comment]
@eveadele @farski it's a RACE!

**Comment by chrisrhoden** [comment]
@eveadele @farski you guys are slow.

**Comment by chrisrhoden** [comment]
yay @eveadele wins

**Pull request merged**